modules/core/shared/src/main/scala/lucuma/core/math/BrightnessUnits.scala

```diff
@@ -19,6 +22,7 @@ object BrightnessUnits {
   trait FluxDensityContinuum[+T]
 
   type BrightnessMeasure[T] = Measure[BigDecimal] Of Brightness[T]
+  type BrightnessMeasureOverTime[T] = NonEmptyMap[Instant, Measure[BigDecimal] Of Brightness[T]]
```

Comment:
I suppose we could use the `BrightnessMeasure` alias here.

Reply:
Sounds like a good idea, to reduce confusion a bit